src/main/java/com/wise23/chariteed/QRCodeGenerator.java

```diff
@@ -0,0 +1,56 @@
+package com.wise23.chariteed;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.Hashtable;
+
+import javax.imageio.ImageIO;
+
+import com.google.zxing.BarcodeFormat;
+import com.google.zxing.EncodeHintType;
+import com.google.zxing.WriterException;
+import com.google.zxing.common.BitMatrix;
+import com.google.zxing.qrcode.QRCodeWriter;
+import com.google.zxing.qrcode.decoder.ErrorCorrectionLevel;
+
+public class QRCodeGenerator {
+
+    private static final int size = 125;
+    private static final String fileType = "png";
+
+    public static void createQRImage(String id) throws WriterException, IOException {
+        // This is still the local url
+        String url = "http://localhost:8080/login/";
```

Comment:
same thing: At least the base of the address (localhost part) should be defined in properties

Reply:
See [above](https://github.com/Miao-99/charite-ed/pull/10#discussion_r1063364605)